### PR TITLE
Create 2022-11-02-incubator-lesson-spotlight-tidyverse-working-with-d…

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

**Comment by MCMaurer** [comment]
Hi @drmowinckels and @RaoulWolf, this PR is for a draft of the Incubator Spotlight blog post for your lesson. If you'd like to review it and provide any feedback, we can get it published as soon as you sign off. Thanks, and let me know if you have any questions!

File: _posts/2022/10/2022-10-25-incubator-lesson-spotlight-tidyverse-working-with-data.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- To be able to effectively alter the shape of data (pivoting)
+- To be able to summarise data efficiently
+
+### Target Audience
+
+Anyone working with data sets that want to learn efficient ways of wrangling and visualising that data with R, using the tidyverse.
+
+## Lesson Progress
+
+This material has been applied during SWC lessons in Oslo 3 times now, using the general state of the materials. Small adjustments are made each time, based on learner feedback, but the overall goals remain the same. Learners are very positive in their feedback of the lesson, and many day they leave with a better understanding of efficient ways of working in R, using the tidyverse.
```
Comment (drmowinckels) [inline _posts/2022/10/2022-10-25-incubator-lesson-spotlight-tidyverse-working-with-data.md]: Learners are very positive in their feedback of the lesson, and many say they leave with a better understanding of efficient ways of working in R, using the tidyverse.

Comment (drmowinckels) [reply _posts/2022/10/2022-10-25-incubator-lesson-spotlight-tidyverse-working-with-data.md]: day to say

File: _posts/2022/10/2022-10-25-incubator-lesson-spotlight-tidyverse-working-with-data.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## How You Can Help Develop This Lesson
+
+We would love to see the content tried by others also and see if the lesson works well for others too. They way we teach it, we always cover each episode (though we might at times skip the later stages of an episode if we fall behind), but each episode is always taught.  We have tried to keep everything detached from any specific field of science, and hope other see this too.
```
Comment (drmowinckels) [inline _posts/2022/10/2022-10-25-incubator-lesson-spotlight-tidyverse-working-with-data.md]: We would love to see the content tried by others also and see if the lesson works well for others too. They way we teach it, we always cover each episode, though we might at times skip the later stages of an episode if we fall behind. We are still a little apprehensive of the `across` lesson, and often find we dont have time for that in two days. It is also quite advanced use (functional programming) and we are uncertain if novices can grasp the concept in such short time. We have tried to keep everything detached from any specific field of science, and hope other see this too.

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Just a couple of small changes to the text, to be more clear.

**Pull request merged**